content/applications/finance/payment_providers.rst

```diff
@@ -169,246 +162,267 @@ Bank payments
-   meant to be used with your test/sandbox credentials, but Odoo generates Sales Orders and Invoices
-   as usual. It isn't always possible to cancel an invoice, and this could create some issues with
-   your invoices numbering if you were to test your payment providers on your main database.
+When you authorize a payment, the funds are reserved on the customer's payment method but they are
+not immediately charged. The charge is only made when you manually capture the payment later on.
+You can also void the authorization to release the reserved funds; this is equivalent to a regular
+cancellation. Capturing payments manually is useful in many situations:
 
-.. _payment_providers/credentials_tab:
+- Receive the payment confirmation and wait until the order is shipped to capture the payment.
+- Review and verify that orders are legitimate before the payment is completed and the fulfillment
+  process starts.
+- Avoid potentially high processing fees for cancelled payments: payment providers will not charge
+  you for voiding an authorization.
+- Hold a security deposit to return later, minus any deductions (e.g., in case of damages).
 
-Credentials tab
-~~~~~~~~~~~~~~~
+To capture the payment after it was authorized, go to the related sales order or invoice and click
+the :guilabel:`Capture Transaction` button. To release the funds, click the :guilabel:`Void
+Transaction` button.
 
-If not done yet, go to the online payment provider website, create an account, and make sure to have
-the credentials required for third-party use. Odoo requires these credentials to communicate with
-the payment provider.
+.. note::
+   - Some payment providers support capturing only part of the authorized amount. The remaining
+     amount can then be either captured or voided. These providers have the value **Full and
+     partial** in the :ref:`table above <payment_providers/online_providers>`. The providers that
+     only support capturing or voiding the full amount have the value **Full only**.
+   - The funds are likely not reserved forever. After a certain time, they may be automatically
+     released back to the customer's payment method. Refer to your payment provider's documentation
+     for the exact reservation duration.
+   - Odoo does not support this feature for all payment providers but some allow the manual capture
+     from their website interface.
 
-The form in this section is specific to the payment provider you are configuring. Please refer to
-the related documentation for more information.
+.. _payment_providers/features/express_checkout:
 
-.. _payment_providers/configuration_tab:
+Express checkout
+================
 
-Configuration tab
-~~~~~~~~~~~~~~~~~
+:ref:`If the payment provider supports this feature <payment_providers/online_providers>`, you can
+allow customers to use the :guilabel:`Google Pay` and :guilabel:`Apple Pay` buttons and pay their
```

Comment:
Not sure if it's worth mentioning but there is also Stripe's Link now (quite anecdotic).